src/main/java/com/minecolonies/coremod/util/AchievementUtils.java

```diff
@@ -38,14 +38,15 @@ public static void syncAchievements(@NotNull final Colony colony)
 
         for (@Nullable final EntityPlayer player : lPlayer)
         {
-            for (final Achievement achievement : colony.getAchievements())
+            for (final Advancement achievement : colony.getAchievements())
             {
-                if (player == null || ModAchievements.achievementGetSupply == achievement)
+                //todo
+                /*if (player == null || ModAchievements.ad == achievement)
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') This block of commented-out lines of code should be removed. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonar.minecolonies.com/coding_rules#rule_key=squid%3ACommentedOutCodeLine)